pom.xml

```diff
@@ -689,6 +689,21 @@
                     <version>1.2.1</version>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <compilerId>javac-with-errorprone</compilerId>
```

Comment:
Master:
```
$ time mvn clean test-compile
...
real	0m16.316s
user	0m38.648s
sys	0m2.888s
```

This PR:
```
$ time mvn clean test-compile
...
real	0m20.876s
user	0m50.651s
sys	0m3.065s
```

On my machine is an extra 4 seconds to compile time.

Master:
```
$ time mvn clean test
...
real	2m43.561s
user	2m16.249s
sys	0m8.279s
```

This PR:
```
$ time mvn clean test
...
real	2m52.186s
user	2m34.531s
sys	0m9.092s
```

And 9 seconds or so for the total build time.

I'm having a hard time figuring out a good way to compare the jars to see if they are actually different.